client/flutter/lib/pages/question_index.dart

```diff
@@ -62,44 +66,79 @@ class _QuestionIndexPageState extends State<QuestionIndexPage> {
               )
             : SliverToBoxAdapter(
                 child: Padding(
-                  padding: const EdgeInsets.all(48.0),
-                  child: CupertinoActivityIndicator(),
-                )
-
-              )
+                padding: const EdgeInsets.all(48.0),
+                child: CupertinoActivityIndicator(),
+              ))
       ],
       title: widget.title,
     );
   }
 
-  Widget _buildQuestion(QuestionItem questionItem) {
-    return Padding(
-      padding: const EdgeInsets.all(8.0),
+  // flutter_html supports a subset of html: https://pub.dev/packages/flutter_html
+}
+
+class QuestionTile extends StatefulWidget {
+  const QuestionTile({
+    @required this.questionItem,
+  });
+
+  final QuestionItem questionItem;
+
+  @override
+  _QuestionTileState createState() => _QuestionTileState();
+}
+
+class _QuestionTileState extends State<QuestionTile>
+    with TickerProviderStateMixin {
+  bool isExpanded;
+  AnimationController rotationController;
+
+  @override
+  void initState() {
+    super.initState();
+    isExpanded = false;
```

Comment:
This bool is never set to its correct value, or used

Reply:
Left over from old setstate way

Reply:
Removing rn